ts/screens/authentication/LandingScreen.tsx

```diff
@@ -222,9 +212,19 @@ class LandingScreen extends React.PureComponent<Props, State> {
     >
       {isDevEnv && <DevScreenButton onPress={this.navigateToMarkdown} />}
 
-      <Content contentContainerStyle={styles.flex} noPadded={true}>
-        <HorizontalScroll cards={this.renderCardComponents()} />
-      </Content>
+      {this.state.isSessionExpired ? (
```

Comment:
just a minor thing, maybe we could avoid using the state and choose directly through the props `props.isSessionExpired` whether to render the component or not, if I understand correctly. What do you think about it?

Reply:
It was my first attempt but it doesn't work: we land on this screen when the session is expired. This screen recognizes that state and shows the info component (before it shows a banner). After that it restores the authentication state to default.

```typescript
if (this.props.isSessionExpired) {
      this.setState({ isSessionExpired: true });
      this.props.resetState();
    }
```

On the previous code it works with props since it shows a timed banner.

In this case we have to show the info component permanently.
If you find an alternative implementation it is welcomed ;)

Reply:
Ops, sorry, I thought the `isSessionExpired` selector always returned the actual state. Then definitely save the initial state and not change the behavior of `isSessionExpired` at this stage!